cmd/embedded.go

```diff
@@ -89,15 +88,12 @@ var (
 		},
 	}
 
-	sections map[string]*section
-	assets   []asset
+	assets []asset
 )
 
 type section struct {
```

Comment:
I think we don't need this struct section.

Reply:
This global variable & struct are not what they look like.

![image](https://user-images.githubusercontent.com/2114189/230903662-2bdaff77-4751-4e15-9ca5-913ede900ed9.png)


![image](https://user-images.githubusercontent.com/2114189/230903393-3d56677b-2ed2-48d3-ad21-56553e93d554.png)

Reply:
Don't know why I cannot send PR to you. Here is a patch.

[refactor.patch](https://github.com/go-gitea/gitea/files/11195803/refactor.patch)

Reply:
I see your point, if you prefer to rewriting the code, then we could do this: 7a8bbeb894f5ed9708df2f9c7be121cfbd29b482

* No need to expose any `asset` field
* No need to make `map[string]*assetfs.LayeredFS`
* No need to `sort.SliceStable`


I didn't do this rewriting because it changes the logic, it isn't similar to "Simplify the AssetsHandlerFunc code", which only deletes unnecessary code.